README.md

```diff
@@ -53,33 +83,66 @@ Install-Package Cogworks.CogFlare
 Add these settings to the **appsettings.json**
 ```js
   "CogFlareSettings": {
+    "IsEnabled": true,
     "ApiKey": "xxx",
     "Email": "xxx@xxx.com",
     "Endpoint": "https://api.cloudflare.com/client/v4/zones/[zoneId]/purge_cache",
-    "KeyNodes": "1234, 031089",
-    "IsEnabled": true,
-    "BlockAliases": "formBlock, newBlock",
-    "Domain": "https://www.example.com"
+    "Domain": "https://www.example.com",
+    "EnableLogging": true, //optional
+    "KeyNodes": "1234, 031089", // optional
+    "BlockAliases": "formBlock, otherFormBlock", // optional
+    "CacheTime": "2592000" // optional => will default to 1 month
+  }
+```
+
+To add cache headers to your pages please add the view component in your Master or required page View
+```razor
+@await Component.InvokeAsync("CacheHeaders")
```

Comment:
If I remember correctly we introduced a constant somewhere in the package not to have to hardcode it (which is prone to errors if the name of the view changes in the future) so it would be the best option to include that one in the documentation 😉